hazelcast-enterprise-kubernetes/README.md

```diff
@@ -179,14 +187,85 @@ spec:
         - name: HZ_DATA
           value: /data/hazelcast
         volumeMounts:
-          - name: hz-persistent-storage
+          - name: hazelcast-storage
             mountPath: /data/hazelcast
       volumes:
-        - name: hz-persistent-storage
+        - name: hazelcast-storage
           persistentVolumeClaim:
             claimName: hz-vc
 ```
 
+### Config Map
+
+The Hazelcast configuration can be stored as ConfigMap and used by the Hazelcast node.
+
+In order to do it, copy `hazelcast.xml` into the `hazelcast-configuration` directory. Then, create a `ConfigMap`.
+
+    $ kubectl create configmap hazelcast-configuration --from-file hazelcast-configuration
+
+Finally, mount ConfigMap into `/data/hazelcast` the same way it was done in case of Persistent Volume. The `hazelcast-service.yaml` file can look as follows:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: hazelcast-ee
+  labels:
+    app: hazelcast-ee
+spec:
+  type: NodePort
+  selector:
+    app: hazelcast-ee
+  ports:
+  - protocol: TCP
+    port: 5701
+    name: hzport
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: hazelcast-ee
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: hazelcast-ee
+    spec:
+      containers:
+      - name: hazelcast-ee-node
+        image: hazelcast/hazelcast-enterprise-kubernetes:3.10
+        ports:
+        - containerPort: 5701
+        livenessProbe:
+          exec:
+            command:
+            - "./liveness.sh"
```

Comment:
we'll have to update this to httpGet :-)

Reply:
done